### PR TITLE
Update SpaceFlower.md

### PR DESCRIPTION
**Pull request by lllz**

### CODE REVIEW

**Comment by lllz** [comment]
Adding some extra conditions (temp and solis). Grammar corrections

File: Biology/SpaceFlower.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
 
 ### Introduction (Background)
 
-Plants in space are plants grown in outer space.
+Plants in the space are the plants grown in the outer space.
```
Comment (Bryukh) [inline Biology/SpaceFlower.md:5]: "in space " (Grammarly)

File: Biology/SpaceFlower.md [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
 They can be used for refreshing the atmosphere for extraterrestrial colonies or
 as a renewable source of food.
 
-And one of the wonders of the Earth nature is flowers.
-They are not only the reproductive structure for many plants but blooms are beautiful.
-Also, flowers in space may provide a psychological benefit to human spaceflight crews.
+One of the wonders of the Earth's nature is flowers.
+They are not only reproductive structures for many plants but also very beautiful blooms.
+Also, flowers in the space may provide a psychological benefit to human spaceflight crews.
```
Comment (Bryukh) [inline Biology/SpaceFlower.md:11]: same - without the

File: Biology/SpaceFlower.md [hunk 3]
```diff
@@ -2,25 +2,25 @@
...
 
 ### Problem
 
-Growing process looks simple but on the Earth.
-In outer space, you have problems with how plants grow without gravity and different types of lighting.
-So for flowers it's more complex tasks.
-How to grow and store flowers in space?
+Growing process looks pretty simple on the Earth.
+In the outer space, you have problems with how plants grow without gravity, temperature, soils and different types of lighting.
+Thus as you can see it's a more complex task for flowers.
```
Comment (Bryukh) [inline Biology/SpaceFlower.md:17]: Thus, as you can see, it's a more complex task for flowers.

**Pull request merged**